From 06bda1b63c2f56e6068c024ad4dec53b90f2296a Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sat, 10 Feb 2018 13:03:48 +0000
Subject: [PATCH] data: Fix broken Optimus with SDDM

This was shown to fix the Xsetup with newer SDDM so that we have a correctly
initialised environment (path, etc) and that we return the exit code of our
helper.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 data/Xsetup.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/data/Xsetup.in b/data/Xsetup.in
index 11d6fae..0e7a0ba 100755
--- a/data/Xsetup.in
+++ b/data/Xsetup.in
@@ -1 +1,2 @@
-@BINDIR@/ldm-session-init
+#!/bin/bash
+exec @BINDIR@/ldm-session-init
-- 
2.16.1

